jetbrains-core/tst/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessageUtilsTest.kt

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sqs.toolwindow
+
+import com.intellij.testFramework.ProjectRule
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Rule
+import org.junit.Test
+import software.amazon.awssdk.services.sqs.model.Message
+import software.amazon.awssdk.services.sqs.model.MessageSystemAttributeName
+
+class PollMessageUtilsTest {
+    @JvmField
+    @Rule
+    val projectRule = ProjectRule()
+
+    @Test
+    fun `Message mapped to columns as expected`() {
+        val table = MessagesTable().apply {
+            tableModel.addRow(message1)
+        }
+
+        assertThat(table.tableModel.items.size).isOne()
+        assertThat(table.tableModel.items.first().body()).isEqualTo("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA")
+        assertThat(table.tableModel.items.first().messageId()).isEqualTo("AAAAAAAAAAA")
+        assertThat(table.tableModel.items.first().attributes().getValue(MessageSystemAttributeName.SENDER_ID)).isEqualTo("1234567890:test1")
+        assertThat(table.tableModel.items.first().attributes().getValue(MessageSystemAttributeName.SENT_TIMESTAMP)).isEqualTo("111111111")
+    }
+
+    @Test
+    fun `Message body truncated as expected`() {
+        // Message 1 has length below 1024
+        val column1 = MessageBodyColumn().valueOf(message1)
+        // Message 2 has length of 1024
+        val column2 = MessageBodyColumn().valueOf(message2)
+        // Message 3 has length of 1025
+        val column3 = MessageBodyColumn().valueOf(message3)
+
+        assertThat(column1?.length).isEqualTo(message1.body().length)
+        assertThat(column2?.length).isEqualTo(MAX_LENGTH)
+        assertThat(column3?.length).isEqualTo(MAX_LENGTH)
+    }
+
+    private companion object {
+        val MAX_LENGTH = 1024
+        val message1: Message = Message.builder()
+            .body("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA")
+            .messageId("AAAAAAAAAAA")
+            .attributes(mapOf(Pair(MessageSystemAttributeName.SENDER_ID, "1234567890:test1"), Pair(MessageSystemAttributeName.SENT_TIMESTAMP, "111111111")))
+            .build()
+        val message2: Message = Message.builder()
+            .body("VJBIGiHnyegmcoGOEqS7EfHB8xU2CTyUHuJOPDap1ThwWFJ3WbdBlj73YGJtWQM8FWYeBxxUzeZOq7kW9WuYQGoFZ923Gm1jz78N9RnX8Z7V8mLMbA32CjBopsmUYKACH3" +
```

Comment:
To make a multiline string, you can use """<string>""". However, in this case it looks like you are making random text, so you can use `RuleUtils.randomName` then pass it the length you want